softlayer/vm/softlayer_vm.go

```diff
@@ -24,7 +24,7 @@ const (
 	softLayerVMtag                 = "SoftLayerVM"
 	ROOT_USER_NAME                 = "root"
 	deleteVMLogTag                 = "DeleteVM"
-	TIMEOUT_TRANSACTIONS_DELETE_VM = 10 * time.Minute
+	TIMEOUT_TRANSACTIONS_DELETE_VM = 60 * time.Minute
```

Comment:
In @posijon's ```has_vm``` Integration test, the delete vm operation is called from ```testhelpers.DeleteVirtualGuest(virtualGuest.Id)``` which doesn't check if the delete vm transaction has started. 
We found that sometimes it takes quite a long time for SL to start the delete vm transaction and have opened a ticket to SL. Currently we believe it's the problem of SL, not the location that runs it. TIMEOUT_TRANSACTIONS_DELETE_VM is increased to avoid the tests fail due to timeout. Actually it doesn't wait 60 minutes but stopping waiting as soon as the delete vm transaction starts.